lib/utilities/attrValidators.js

```diff
@@ -0,0 +1,99 @@
+/**
+ * More validators for class attributes. 
+ * @class AttrValidators
+ * @static
+ */
+var AttrValidators = {
+
+  /**
+   * Returns a function that checks if a value is an instance of the given
+   * contructor isInstanceOf.
+   *
+   * @method isInstanceOf
+   * @param ctor {Object} The contructor function
+   */
+  isInstanceOf: function(ctor) {
+    if (!ctor) {
+      console.error(ctor);
+      console.error('Constructor object to check against was falsy.');
+      return false;
+    }
+
+    return function (val, attrName) {
+      return (val instanceof ctor);
+    };
+  },
+
+  /**
+   * Returns a function that checks if a value is in the given constants object.
+   *
+   * @method isValueIn
+   * @param obj {Object} The object of constants (key integer pairs)
+   */
+  isValueIn: function (obj) {
```

Comment:
Maybe this should be renamed `isValueInEnum` if it's just for "key integer pairs". But it seems like it would work for other types of objects besides enums, too.